docker/run-as-loxi

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash -e
+
+loxi_uid=$(stat -c'%u' /loxi/loxigen.py)
+loxi_gid=$(stat -c'%g' /loxi/loxigen.py)
+usermod -u $loxi_uid -g $loxi_gid loxi || echo "ok"
```

Comment:
Maybe I'll leave the usermod version if it's ok with you - I slightly prefer having a user defined with the UID, even if not strictly necessary.